### ISSUE TITLE
[A11y Compliance] SfProductCard should have alt text 

### ISSUE THREAD

**Issue by mayashavin** [DivanteLtd/storefront-ui#113]
<img width="551" alt="Screen Shot 2019-05-25 at 23 11 17" src="https://user-images.githubusercontent.com/6650139/58374179-8d7f2580-7f42-11e9-8d3d-ea5c4a88fa21.png">
It should take `title` value and assign it to `alt` tag in addition.

**Comment by filrak** [comment]
done with SfLoader
